core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/LimsUtils.java

```diff
@@ -650,25 +651,29 @@ public static boolean isPlainSample(Sample sample) {
 
   public static boolean isIdentitySample(Sample sample) {
     if (!isDetailedSample(sample)) return false;
-    return sample instanceof Identity;
+    return sample instanceof Identity || ((DetailedSample) sample).getSampleClass().getSampleCategory().equals(Identity.CATEGORY_NAME);
```

Comment:
What's the correct behaviour if null? I think it's neither true nor false, so I'm happy with an NPE.